lib/bolt/config/transport/base.rb

```diff
@@ -116,24 +119,21 @@ def self.options
           assert_type
         end
 
-        # Validates that each option is the correct type. Types are loaded from the OPTIONS hash.
+        # Validates that each option is the correct type. Types are loaded from the TRANSPORT_OPTIONS hash.
         private def assert_type
           @config.each_pair do |opt, val|
```

Comment:
This automated validation is so nice, it'd be nice to use it for general Bolt config as well (in another ticket!)

Reply:
That's the plan! :D